src/components/App/App.tsx

```diff
@@ -13,29 +13,28 @@ interface IState{
     name: string
     population: number
   }[],
-  questions:Array<string> | string
   currentCountry: {
     name: string
+    population: number
+    capital: string
+    borders: string[]
```

Comment:
As we add more properties to our country objects, we need to remember to keep updating these iStates as we go!